### PR TITLE
Change documentation about maintainer alumnus status field

### PR DESCRIPTION
**Pull request by kytrinyx**
The linter chokes if given a string here. I'm not sure if the linter
changed, or if this never worked in the first place as documented.

### CODE REVIEW

**Review by wolf99 — APPROVED** [review]
LGTM

**Comment by iHiD** [comment]
We do accept strings in the website code, but I prefer that we don't. I believe we decided to allow it in case there was someone who was uncomfortable with the word alumnus, but I suggest we just merge this for now, and address it if we ever hit that scenario.

File: maintaining-a-track/maintainer-configuration.md [hunk 1]
```diff
@@ -28,11 +28,7 @@ Being featured on the Exercism website is entirely optional, and we're generatin
 ## Alumnus
 
 When you decide to move on, submit and merge a PR that sets the `alumnus` key
-for your maintainer entry to one of the following strings:
-`alumnum`, `alumnus` or `alumna`.
-This is the string that will appear with your bio on the website. These terms
-have been chosen to enable you to choose the one you feel most appropriate for
-your gender.
+for your maintainer entry to false.
```
Comment (petertseng) [inline maintaining-a-track/maintainer-configuration.md:31]: Thinking about it in non-computer terms, it seems weird that moving on means you decide to not be an alumnus (set `alumnus` to false `false`). Would it thus be better to use `true` here?

Comment (iHiD) [reply maintaining-a-track/maintainer-configuration.md:31]: Good catch. This should say true.

**Pull request closed without merging**